README.md

```diff
@@ -63,12 +63,17 @@ attribute|default|description
 ---------|-------|-----------
 caseSensitive|false|Indicates whether comparisons should be case sensitive.
 className|null|give a custom class name to the root element
+width|430|width of the fuzzy searchbox
+inputStyle|{}|Styles passed directly to the `input` element.
+inputWrapperStyle|{}|Styles passed directly to the `input` wrapper `div`.
+resultsStyle|{}|Styles passed to each `result` in the dropdown list.
+resultsWrapperStyle|{}|Styles passed directly to the dropdown wrapper.
+selectedResultStyle|{}|Styles passed directly to current 'active' item.
```

Comment:
Can we use more generic term like `list` instead of `result`? Sometimes they might be suggestions instead of results so U think it makes sense to have more generic names.

Reply:
Updated 🙇 

- `resultsStyle` -> `listItemStyle`
- `selectedResultStyle` -> `selectedListItemStyle`
- `resultsWrapperStyle` -> `listWrapperStyle`